File: app/src/main/java/org/droidupnp/model/upnp/DeviceDiscoveryObserver.java
package org.droidupnp.model.upnp;

import com.m3sv.droidupnp.data.UpnpDeviceEvent;

public interface DeviceDiscoveryObserver {

    void addedDevice(UpnpDeviceEvent device);

    void removedDevice(UpnpDeviceEvent device);
}

File: app/src/main/java/org/droidupnp/model/upnp/DeviceDiscoveryObserver.kt
package org.droidupnp.model.upnp

import com.m3sv.droidupnp.data.UpnpDeviceEvent

interface DeviceDiscoveryObserver {
    fun addedDevice(device: UpnpDeviceEvent)

    fun removedDevice(device: UpnpDeviceEvent)
}
